packages/animations/lib/src/open_container.dart

```diff
@@ -29,6 +29,10 @@ enum ContainerTransitionType {
   fadeThrough,
 }
 
+/// Callback function which is called when the [OpenContainer]
+/// is closed.
+typedef onClosedCallback<S> = void Function(S data);
```

Comment:
Please follow our style guide for naming: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#naming-rules-for-typedefs-and-function-variables